### DISCUSSION TITLE
Hide achievements in Assets List

### DISCUSSION THREAD

**Discussion by ghost** [RetroAchievements/RAIntegration#1123] [Ideas]
Something for hiding would be useful, when i was devving for a game it was hard for me to navigate through 60 achevements to find what i want... Maybe some category in local-unofficial-official category? It would be usefull for games with more achievements too

**Comment by Jamiras** [comment]
I don't understand the request. If you're having trouble finding things, you can sort by clicking on the headers

**Reply by ghost** [reply]
Yes i can, but it was still hard for me to navigate through them. I thought about having an option to hide published achievements in assets list. Something like a new category that i can drop exact achievements i want.

So that achievement is still there, but just hidden. Waiting for unhidding when needed.
